EasyInventory/src/main/java/com/easyinventory/enterprise/controller/EasyInventoryController.java

```diff
@@ -43,7 +43,7 @@
     @RequestMapping(value="/category/newcategory", method = RequestMethod.POST)
     public List<Category> addCategory(@RequestBody Category category){
         categoryRepository.saveAndFlush(category);
-        List<Category> categoryList = new ArrayList<>();
```

Comment:
Variable "new  ArrayList<>()" is redundant

Reply:
Good catch... no need to initialize it, when it is about to be reassigned in the next line.